scripts/exgdas_enkf_ecen.sh

```diff
@@ -209,11 +208,6 @@ if [ ${SUFFIX} = ".nc" ]; then
       LEVS_ENKF=${LEVS_ENKF:-$($NCLEN atminc_ensmean lev)} # get LEVS
    fi
    JCAP_ENKF=${JCAP_ENKF:--9999} # there is no jcap in these files
-else
-   LONB_ENKF=${LONB_ENKF:-$($NEMSIOGET atmanl_ensmean dimx | awk '{print $2}')}
-   LATB_ENKF=${LATB_ENKF:-$($NEMSIOGET atmanl_ensmean dimy | awk '{print $2}')}
-   LEVS_ENKF=${LEVS_ENKF:-$($NEMSIOGET atmanl_ensmean dimz | awk '{print $2}')}
-   JCAP_ENKF=${JCAP_ENKF:-$($NEMSIOGET atmanl_ensmean jcap | awk '{print $2}')}
```

Comment:
If you are going to get rid of the nemsio code, why not just get rid of the whole if?

Reply:
I was just talking about this particular `if` (that the `else` was attached to). Agree removing all the nemsio stuff is a separate task.

Reply:
done.